packages/vkui/src/components/WriteBar/WriteBar.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { Icon28SmileOutline } from '@vkontakte/icons';
+import { CanvasFullLayout, DisableCartesianParam } from '../../storybook/constants';
+import { getFigmaPage } from '../../storybook/helpers';
+import { WriteBarIcon } from '../WriteBarIcon/WriteBarIcon';
+import { WriteBar, WriteBarProps } from './WriteBar';
+
+const story: Meta<WriteBarProps> = {
+  title: 'Blocks/WriteBar',
+  component: WriteBar,
+  parameters: { ...CanvasFullLayout, ...getFigmaPage('UsersStack'), ...DisableCartesianParam },
```

Comment:
```suggestion
  parameters: { ...CanvasFullLayout, ...getFigmaPage('WriteBar'), ...DisableCartesianParam },
```

как на счёт для компонент, которые хранятся на одной и той же борде, всё-равно завести свой ключ, а ссылку переиспользовать ? А то кажется как-будто перепутали))

Reply:
Если честно, то пока не понятно, что делать с фигмой - к ней самой доступ есть только у нас (у остальных сломается), ну и ссылки на элементы меняются 😞

Reply:
Может пока что вообще фигму не выводить?